packages/webdriverio/src/commands/browser/switchWindow.ts

```diff
@@ -64,6 +64,14 @@ export default async function switchWindow (
         if (matchesTarget(title)) {
             return tab
         }
+
```

Comment:
rename `urlOrTitleToMatch` to something more logical if the windowName is now also part of this.

Reply:
Will change to matcher ,